corehq/apps/hqwebapp/tests/test_requirejs.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+import logging
+import os
+import re
+import subprocess
+
+from django.test import SimpleTestCase
+
+logger = logging.getLogger(__name__)
```

Comment:
the last time i tried to set up a logger, it seemed to be a bit more difficult than this. there were some configs in settings.py that had to be added. if this is working as is, then just ignore <<

Reply:
I did add that last minute, so...let's see if tests pass.

Reply:
You're right - doesn't interfere with the test itself, but the output isn't visible. Updated to stuff all of the output into the assertion failure. (Also moved some code around because I'm in the middle of adding more tests to this file.)